src/Events/Admin/Settings/Community_Upsell.php

```diff
@@ -48,6 +56,48 @@ public function add_actions(): void {
 		add_action( 'tribe_settings_do_tabs', [ $this, 'add_tab' ] );
 	}
 
+	/**
+	 * Add filters.
+	 *
+	 * @since TBD
+	 */
+	public function add_filters(): void {
+		add_filter( 'tribe_settings_form_class', [ $this, 'filter_tribe_settings_form_classes' ], 10, 2 );
+		add_filter( 'tribe_settings_no_save_tabs', [ $this, 'filter_tribe_settings_no_save_tabs' ] );
+	}
+
+	/**
+	 * Adds a class to the settings form to allow for custom styling.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $classes
+	 */
+	public function filter_tribe_settings_form_classes( $classes ): array {
+		if ( ! in_array( 'tec-settings__community-tab--active', $classes ) ) {
+			return $classes;
+		}
+
+		$classes[] = 'tec-events-settings__upsell-form';
+
+		return $classes;
+	}
+
+	/**
+	 * Adds the Community tab to the list of tabs that should not be saved.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $tabs
+	 *
+	 * @return array
+	 */
+	public function filter_tribe_settings_no_save_tabs( $tabs ): array {
+		$tabs[] = $this->slug;
+
+		return $tabs;
+	}
+
```

Comment:
Just a note that these new methods should be below the class fields (the `$template` field can be seen just below this line).

Reply:
That's...weird. I'll move the field up where it belongs